framework/tests/returnsreceiverlombok/BuilderTest.java

```diff
@@ -0,0 +1,26 @@
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Builder
+@Accessors(fluent = true)
+public class BuilderTest {
+    @Getter @Setter private Integer x;
+    @Getter @Setter @NonNull private Integer y;
+    @Getter @Setter @NonNull private Integer z;
+
+    public static void test_simplePattern() {
+        BuilderTest.builder().x(0).y(0).build(); // good builder
+        BuilderTest.builder().y(0).build(); // good builder
```

Comment:
These two builders are not "good" - they're both incorrect. That comment looks like a holdover from the Object Construction Checker, and should be removed.